digitalstrategie/assets/scss/components/_ds-block-overwrites.scss

```diff
@@ -5,11 +5,15 @@
         margin: auto !important;
     }
 
+    .ds-hero-no-image {
+        margin-bottom: 5 * $spacer !important;
+        text-align: center;
+    }
+
     // overwrite of herounit text box
     .ds-hero-image__overlay {
+        @extend .ds-hero-no-image;
```

Comment:
as @extend overwrites any other styling you add if it clashes with the extended styling I would more inclined to add the `.ds-hero-image__overlay`class to the styling above so have 

```
.ds-hero-no-image,
 ds-hero-image__overlay {
        margin-bottom: 5 * $spacer !important;
        text-align: center;
    }

    // overwrite of herounit text box
    .ds-hero-image__overlay {
```
but as it's more a coding syntax choice so easy either way